src/types/NodeType.ts

```diff
@@ -0,0 +1,343 @@
+/* eslint-disable no-underscore-dangle */
+
+import {
+  GraphQLFieldConfig,
+  GraphQLFieldConfigMap,
+  GraphQLFieldResolver,
+  GraphQLObjectType,
+  GraphQLObjectTypeConfig,
+  GraphQLResolveInfo,
+  GraphQLString,
+  Thunk,
+} from 'graphql';
+import {
+  connectionDefinitions,
+  fromGlobalId,
+  globalIdField,
+  nodeDefinitions,
+} from 'graphql-relay';
+import invariant from 'invariant';
+import camelCase from 'lodash/camelCase';
+import snakeCase from 'lodash/snakeCase';
+import pluralize from 'pluralize';
+import { Class } from 'utility-types';
```

Comment:
Leveraged a flow to typescript converter and it outputted this for `$Class` types. Happy to remove if you know the correct way

Reply:
we already use `utility-types` so this seems reasonable to me